docs/pages/code/collatz-conjecture.php

```diff
@@ -12,17 +12,13 @@
 use loophp\collection\Collection;
 
 // The Collatz conjecture (https://en.wikipedia.org/wiki/Collatz_conjecture)
-$collatz = static function (int $value): int {
-    return 0 === $value % 2 ?
-        $value / 2 :
-        $value * 3 + 1;
-};
+$collatz = static fn (int $value): array => 0 === $value % 2
+    ? [$value / 2]
+    : [$value * 3 + 1];
 
-$c = Collection::unfold($collatz, 25)
-    ->until(
-        static function ($number): bool {
-            return 1 === $number;
-        }
-    );
+$collection = Collection::unfold($collatz, 10)
+    ->unwrap()
+    ->normalize()
+    ->until(static fn ($number): bool => 1 === $number);
```

Comment:
We could simplify this as:

```php
$collection = Collection::unfold($collatz, 10)
    ->unwrap()
    ->until(static fn (int $number): bool => 1 === $number);
```

Reply:
right, that works now because `all` normalizes by default

Reply:
:+1: